### PR TITLE
Sifu 0.1.1 new formula

### PR DESCRIPTION
**Pull request by vesin**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

**Comment by vesin** [comment]
Can i get a comment what is wrong with the formula? where to put command-line.jar ?

File: Formula/sifu.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Sifu < Formula
+  desc "Installation of Sifu tool."
+  homepage "https://codesifu.com"
+  url "https://codesifu.com/download/sifu-0.1.1.tar.gz"
+  sha256 "d0ef00a475766bf4717b4721044bdbd0f1c8c02be3a1f824738d306b08a06cd2"
+
+  depends_on :java # sifu runs on JVM
```
Comment (MikeMcQuaid) [inline Formula/sifu.rb]: Can remove this comment.

File: Formula/sifu.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class Sifu < Formula
+  desc "Installation of Sifu tool."
```
Comment (MikeMcQuaid) [inline Formula/sifu.rb]: Please describe what the tool does, not the formula.

**Comment by MikeMcQuaid** [comment]
> where to put command-line.jar ?

Anywhere except `bin` or `lib`.

**Pull request closed without merging**